02-peer-protocol.md

```diff
@@ -163,17 +163,17 @@ The `NodeStateUpdate` message is used to tell a peer about a change in the node
     `repeated string pairs = 2`
     The sender's list of trading pair symbols, constructed with the base currency first, followed by a  '/' separator and the quote currency (e.g., [“LTC/BTC”, “DAI/BTC”])
 
-    `string raiden_address = 3`
-    The sender's Raiden address
+    `string connext_address = 3`
+    The sender's connext address
```

Comment:
I think this should be named connext public key. Example: `indra123abc`

Reply:
Fixed.